tests/functional/UseCase/Project/CreateDropProjectTest.php

```diff
@@ -132,7 +132,7 @@ public function testCreateProject(): void
         foreach ($policy['bindings'] as $binding) {
             if ($binding['role'] === 'roles/storage.objectAdmin') {
                 $key = array_search('serviceAccount:' . $publicPart['client_email'], $binding['members']);
-                if ($key) {
+                if ($key !== false) {
```

Comment:
ked tam bol len jeden zaznam tak to nezmazalo, bo to je key a vtedy je key 0 takze to failovalo 🤦